packages/core/saved-objects/core-saved-objects-migration-server-internal/src/model/model.ts

```diff
@@ -113,18 +117,20 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
           // index
           sourceIndex: Option.none,
           targetIndex: `${stateP.indexPrefix}_${stateP.kibanaVersion}_001`,
-          sourceIndexMappings: indices[source].mappings,
+          sourceIndexMappings: indices[source!].mappings,
           // in this scenario, a .kibana_X.Y.Z_001 index exists that matches the current kibana version
           // aka we are NOT upgrading to a newer version
           // we inject the target index's current mappings in the state, to check them later
-          targetIndexCurrentMappings: indices[source].mappings,
+          targetIndexCurrentMappings: indices[source!].mappings,
           targetIndexMappings: mergeMigrationMappingPropertyHashes(
             stateP.targetIndexMappings,
-            indices[aliases[stateP.currentAlias]!].mappings
+            indices[source!].mappings
           ),
           versionIndexReadyActions: Option.none,
         };
-      } else if (
+      }
+
+      if (
```

Comment:
Yes, sorry, I made this change because I found the code more readable in an IDE in the collapsed state without nested `else if`s at multiple levels. I'll change it back!